### DISCUSSION TITLE
A comment on deepkit.

### DISCUSSION THREAD

**Discussion by Lydanne** [marcj/deepkit#226] [General]
I hope the deepkit team can create a warehouse for resource summary,  be similar to > https://github.com/nestjs/awesome-nestjs

Here is a recommended deepkit template project to quickly build a development project.

https://github.com/WumaCoder/deepkit-template.git

Welcome to star, deepkit cowhide👍.

**Comment by marcus-sa** [comment]
https://github.com/convoy-js/convoy
WIP framework for building microservices using an event-driven architecture
- DDD
- CQRS
- Event Sourcing
- etc...
